ha_test.go

```diff
@@ -39,72 +40,66 @@ func TestHA_Integration(t *testing.T) {
 	assert.Equal(t, "commons._events", (res.Collections)[0])
 }
 
-func TestHA_OK_FirstFastest(t *testing.T) {
-	ctx := testCtx()
-
+func (s *HaSuite) TestHA_OK_FirstFastest() {
 	f0 := newFakeQuerier("0", time.Millisecond, nil)
 	f1 := newFakeQuerier("1", 2*time.Millisecond, nil)
 
 	ha := rockset.NewHA(f0, f1)
 
-	res, errs := ha.Query(ctx, "SELECT 1")
-	require.Nil(t, errs)
-	assert.Equal(t, "0", *res.QueryId)
+	res, errs := ha.Query(s.ctx, "SELECT 1")
+	s.Nil(errs, errs)
+	// require.Nil(t, errs)
+	s.Equal("0", *res.QueryId)
+	// assert.Equal(t, "0", *res.QueryId)
 }
 
-func TestHA_OK_SecondFastest(t *testing.T) {
-	ctx := testCtx()
-
+func (s *HaSuite) TestHA_OK_SecondFastest() {
 	f0 := newFakeQuerier("0", 2*time.Millisecond, nil)
 	f1 := newFakeQuerier("1", time.Millisecond, nil)
 
 	ha := rockset.NewHA(f0, f1)
 
-	res, errs := ha.Query(ctx, "SELECT 1")
-	require.Nil(t, errs)
-	assert.Equal(t, "1", *res.QueryId)
+	res, errs := ha.Query(s.ctx, "SELECT 1")
+	s.Nil(errs, errs)
+	s.Equal("1", *res.QueryId, "Query Id %d != 1", *res.QueryId)
 }
 
-func TestHA_OK_FirstFails(t *testing.T) {
-	ctx := testCtx()
-
+func (s *HaSuite) TestHA_OK_FirstFails() {
 	f0 := newFakeQuerier("0", time.Millisecond, errors.New("failed"))
 	f1 := newFakeQuerier("1", 2*time.Millisecond, nil)
 
 	ha := rockset.NewHA(f0, f1)
 
-	res, errs := ha.Query(ctx, "SELECT 1")
-	require.Nil(t, errs)
-	assert.Equal(t, "1", *res.QueryId)
+	res, errs := ha.Query(s.ctx, "SELECT 1")
+	s.Nil(errs, errs)
+	s.Equal("1", *res.QueryId, "Response Query Id %d != 1", *res.QueryId)
+	s.Assert().Equal("1", *res.QueryId)
+	// assert.Equal(t, "1", *res.QueryId)
```

Comment:
remove

Reply:
Good catch, thought I had scrubbed that.
What do you think, Go linter for commented code?